docs/getting-started/continuous_integration.md

```diff
@@ -136,6 +135,50 @@ jobs:
 {% endraw %}
 ```
 
+#### Branches and paths
+
+You can make CIFuzz to trigger only on certain branches or paths by following the
+instructions [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).
+For example, the following code can used to trigger CIFuzz only on C/C++ code
+residing on master and release branches:
+
+```yaml
+name: CIFuzz
+on:
+  pull_request:
+    branches: 
+      - master
+      - 'releases/**'
+    paths:
+      - '**.c'
+      - '**.cc'
+      - '**.cpp'
+      - '**.cxx'
+      - '**.h'
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+     with:
+       oss-fuzz-project-name: 'example'
+       dry-run: false
```

Comment:
I wonder if dry-run is mandatory, it seems like it would be better as an optional param that is false by default.
(Thought this isn't really a discussion for this PR).

Reply:
I dont know this.